### ISSUE TITLE
Will this work with off-the-shelf AP?

### ISSUE THREAD

**Issue by vbogach** [seemoo-lab/nexmon_csi#8]
I wanted to use Atheros CSI tool initially, but it appears that is needs to be installed both on the AP and the device. Will this software work installed only on the device?

**Comment by jlinktu** [comment]
Yes, you only have to install it on the device on that you want to extract the CSI.
